packages/block-library/src/spacer/controls.js

```diff
@@ -44,10 +44,6 @@ function DimensionInput( { label, onChange, isResizing, value = '' } ) {
 		defaultValues: { px: 100, em: 10, rem: 10, vw: 10, vh: 25 },
 	} );
 
-	const handleOnChange = ( unprocessedValue ) => {
-		onChange( unprocessedValue.all );
-	};
```

Comment:
This handler creates a bug if used with both controls. There is no `all` property on the value passed by `UnitControl`. I think this may have been a problem previously (it hasn’t been since the `UnitControl` is never shown). I didn’t search for issues.